### ISSUE TITLE
Feedback on 40f36ca.

### ISSUE THREAD

**Issue by Viech** [Unvanquished/updater#3]
General layout/size/proportions are looking good!

This feedback is mainly based on "what other launchers do":

- [ ] There are two empty toolbars in the launcher window.
- [ ] You don't need to display anything above the download bar as the bar has enough space for all relevant data. It could read "Updating 200/564 MiB (3.2 MiB/s ↓ 0 MiB/s ↑ 5:23 left)" for example. Think of the bar as the only way to communicate with the user and you will automatically strip your output down to just the currently relevant info.
- [ ] In particular, don't display the install location on the launcher window. You can have a popup window asking for an install location at first start, after that the choice would just be remembered.
- [ ] The launcher errors out with "Install dir does not exist. Please select another". Of course a folder named unvanquished doesn't exist when you want to install Unvanquished, this doesn't mean the installation should fail… :stuck_out_tongue: 
- [ ] In case there is an actual error with the selected download directory, it would be most convenient to re-open the install location selection dialogue. Make sure the status/progress bar is visible and shows the reason for the failure (alternatively display the reason in the dialogue).
- [ ] Except for when the popup window is open on first launch, the launcher should always automatically start to download/update. The play button is the only button that the user should ever press on any subsequent launch. (And this is only true because users could want to read the news!)

**Comment by mjstahlberg** [comment]
Unticked the second and third point as they aren't actually implemented.

**Comment by DolceTriade** [comment]
I disagree with the third point. The install location will disappear once the update starts.

**Comment by mjstahlberg** [comment]
Having an additional status bar disappear after a while just doesn't look good. Try to put all information you want to tell the user into the download/status bar. If the download location would show up there before the download starts (which it would do nearly immediately anyway), fine with me. If you want to show more than what fits in there you're likely trying to show too much anyway. :wink:

**Comment by DolceTriade** [comment]
Fair enough. I likely won't get around to it for a while though since QT UIs are annoying. As for the moving part, it is also lower priority.
